cmd/minishift/cmd/delete.go

```diff
@@ -40,8 +40,6 @@ func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
-	setSubcriptionManagerParameters()
-
```

Comment:
@hferentschik yes we don't have username/password flag for stop/delete any more but that doesn't effect our rhel registratrar once we have #487 implemented then take that details from there and use it for delete/stop. We can create a issue for same just to track.